src/core_ocean/analysis_members/mpas_ocn_mixed_layer_depths.F

```diff
@@ -216,9 +221,16 @@ subroutine ocn_compute_mixed_layer_depths(domain, timeLevel, err)!{{{
          call mpas_pool_get_subpool(block % structs, 'state', statePool)
          call mpas_pool_get_subpool(block % structs, 'mesh', meshPool)
          call mpas_pool_get_subpool(block % structs, 'diagnostics', diagnosticsPool)
+         call mpas_pool_get_subpool(block % structs, 'forcing', forcingPool)
          call mpas_pool_get_subpool(block % structs, 'mixedLayerDepthsAM', mixedLayerDepthsAMPool)
          call mpas_pool_get_subpool(statePool, 'tracers', tracersPool)
 
+         !Use work array for storing of pressure, potentially modified by landIce
+         call mpas_pool_get_subpool(block % structs, 'mixedLayerDepthsAMScratch', scratchPool)
+         call mpas_pool_get_field(scratchPool, 'workArrayPressure', workArrayField)
+         call mpas_allocate_scratch_field(workArrayField, .true.)
+         workArray => workArrayField % array
```

Comment:
I know this is picky, but could you rename `workArray*` to `pressureAdjustedLandIce` or something else that's more readable?

Reply:
not a problem.  Sounds like a good change for clarity.